NameGame/NameGame/Services/ImageService.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ImageService.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/14/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class ImageService {
+    
+    let sharedCache = NSCache<NSString, UIImage>()
```

Comment:
So we won't want a cache here. This service should only be responsible for fetching images over a network.